core/station/impl/src/core/assets.rs

```diff
@@ -19,5 +19,26 @@ thread_local! {
             name: "Ethereum".to_string(),
             metadata: Metadata::default(),
         },
+        Asset {
+            blockchain: Blockchain::Ethereum,
+            standard: BlockchainStandard::ERC20,
+            symbol: "ETH".to_string(),
+            name: "Ethereum".to_string(),
+            metadata: Metadata::default(),
+        },
```

Comment:
ETH is not erc20

Reply:
I guess we want something like USDC here. We have to pass USDC address somewhere here. And specify that network type is "testnet"